website/search/search.py

```diff
@@ -61,6 +61,23 @@ def update_preprint(preprint, index=None, bulk=False, async_update=True, saved_f
         index = index or settings.ELASTIC_INDEX
         return search_engine.update_preprint(preprint, **kwargs)
 
+@requires_search
+def update_group(group, index=None, bulk=False, async_update=True, saved_fields=None, deleted_id=None):
```

Comment:
Passing in deleted_id here, because currently when an OSFGroup is deleted, it's literally deleted, not marked as deleted.